app/src/modules/analytics/events/extension.ts

```diff
@@ -17,6 +18,19 @@ interface EventBatch {
 const sendEventsBatch = (eventBatch: EventBatch): void => {
   eventBatch.events.forEach((event) => {
     const eventConfig = { time: event.eventTs };
+
+    /* ADDING EXTRA INFO FOR RULE EXECUTION EVENTS */
+    if (event.eventName === RULES.RULE_EXECUTED) {
+      const currentUserId = window.uid;
+      const ruleCreator = event.eventParams?.["rule_creator"];
+
+      if (ruleCreator && currentUserId) {
```

Comment:
If `ruleCreator` or `currentUserId` is not present then `eventParams` will contain `rule_creator`. Do we need to send `rule_creator`?

Reply:
no we don't. Will move the deletion outside the `if` scope 👍🏻